### ISSUE TITLE
Playground block: A11Y: Unlabeled button in code editor

### ISSUE THREAD

**Issue by alexstine** [WordPress/playground-tools#286]
Test page: https://learn.wordpress.org/test/wordpress-playground-block-plugin-test-page/

There is an unlabeled button with icon only, not sure what it should be called.

```
<button type="button" class="components-button file-tab file-tab-extra is-secondary"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M18 11.3l-1-1.1-4 4V3h-1.5v11.3L7 10.2l-1 1.1 6.2 5.8 5.8-5.8zm.5 3.7v3.5h-13V15H4v5h16v-5h-1.5z"></path></svg></button>
```

**Comment by alexstine** [comment]
Fixed, thanks!
